.github/workflows/test.yml

```diff
@@ -6,15 +6,31 @@ on:
       - develop
   pull_request: {}
 
+concurrency: 
+  group: ${{ github.head_ref }}
```

Comment:
This should include the pipeline, otherwise it's concurrency across all CI workflows that use the head ref - e.g. https://github.com/holochain/holochain/blob/develop/.github/workflows/test.yml#L10

That uses `github.workflow` to keep workflow concurrency separate

Reply:
That or you can just use `github.ref_name`. https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context

Reply:
Yes, but I think probably both? I think either `head_ref` or `ref_name` will conflict if you put them into two different workflows

Reply:
updated -- I just copied the settings from the holochain workflow.